go.mod

```diff
@@ -4,16 +4,16 @@ go 1.24.2
 
 require (
 	dario.cat/mergo v1.0.2
-	github.com/evanphx/json-patch v0.5.2
+	github.com/evanphx/json-patch v5.9.11+incompatible
```

Comment:
Weird...

- https://github.com/evanphx/json-patch/issues/98
- https://github.com/evanphx/json-patch/issues/145
- https://github.com/evanphx/json-patch/issues/98
- https://github.com/evanphx/json-patch/issues/102
- https://github.com/evanphx/json-patch/releases

May just revert this for now unless I can get the v5 correctly pulled. 0.5.2 is from 2021

Reply:
Afaik, `+incompatible` is just a cosmetic 'error' related to how the repo manages their tags.